### PR TITLE
docs: dent releases, members

### PR DESCRIPTION
**Pull request by royalpinto007**
The homepage lacked many details of the latest releases, so I made sure to include them along with information about the members and an architecture diagram.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,13 +6,40 @@ nav_order: 1
...
 
-With the second release, Beeblebrox, DENT now includes secure scaling with IPv6 and NAT.
+## Releases
 
-In the third release, Cynthia, developers promise a rapid release cycle that will implement Traffic Control Persistence, QoS, IGMP Policing, and Egress Policing.
+The ["Arthur" release (v1)](https://github.com/dentproject/dentOS/wiki/1.0-Release-notes), launched in Q4 2020, introduced fundamental features such as L2 Bridge (vlan aware), IPv4 forwarding, Static Routing, Dynamic Routing – BGP, and more. It provided a robust foundation for network configuration and management, enabling seamless integration into decentralized enterprise environments.
+
+In Q1 2022, the ["Beeblebrox" release (v2)](https://github.com/dentproject/dentOS/milestone/2) brought significant enhancements to DENT. Notable additions included NAT-44 and PT, NAT on pure Layer-3 port, support for configuring 802.1q interfaces, PoE controller integration, Dynamic Routing support, and Scale & Performance Improvements. These advancements fortified DENT's capabilities in handling diverse networking requirements with improved efficiency and scalability.
```
Comment (taskin0003) [inline index.md:17]: what happened to IPv6?

File: index.md [hunk 2]
```diff
@@ -6,13 +6,40 @@ nav_order: 1
...
+
+The "Cynthia" release series marked a significant milestone for DENT. With the [v3.0 release](https://github.com/dentproject/dentOS/milestone/3) in Q1 2023, DENT introduced TC Persistence (Petunia), a new Kernel 5.15 (LTS), and support for IEEE 802.1x, QoS, IPv6, IGMP Snooping, and Egress Policer. These features enhanced DENT's security, performance, and compatibility, positioning it as a robust solution for distributed enterprise edge networking.
+
+The subsequent updates in the "Cynthia" series further solidified DENT's capabilities. The [v3.1 release](https://github.com/dentproject/dentOS/releases/tag/v3.1) in Q2 2023 extended test coverage for QoS, ACL, and Bridging/VLAN, ensuring comprehensive validation of these critical features.
+
+Additionally, the [v3.2 release](https://github.com/dentproject/dentOS/releases/tag/v3.1) in Q3 2023 included an update to Debian 10, introduced DentOS VM for enhanced virtualization support, and extended test coverage for IPV6, IGMP Snooping, Egress Policer, Storm Control, and Devlink integration. These updates reflected DENT's commitment to continuous improvement and adaptation to evolving networking needs.
```
Comment (taskin0003) [inline index.md:23]: The link is wrong. It should have been https://github.com/dentproject/dentOS/releases/tag/v3.2

File: index.md [hunk 3]
```diff
@@ -6,13 +6,40 @@ nav_order: 1
...
 
-In the first release, Arthur, DENT utilizes the Linux 5.6 Kernel to leverage switchDev, enabling the replacement of proprietary SDKs and NOSes with a standard open Linux interface. It also supports existing Linux toolchains.
+![SwitchDev based Open Networking Operating System (NOS)](Images/DENT-Architecture.png)
 
-With the second release, Beeblebrox, DENT now includes secure scaling with IPv6 and NAT.
+## Releases
 
-In the third release, Cynthia, developers promise a rapid release cycle that will implement Traffic Control Persistence, QoS, IGMP Policing, and Egress Policing.
+The ["Arthur" release (v1)](https://github.com/dentproject/dentOS/wiki/1.0-Release-notes), launched in Q4 2020, introduced fundamental features such as L2 Bridge (vlan aware), IPv4 forwarding, Static Routing, Dynamic Routing – BGP, and more. It provided a robust foundation for network configuration and management, enabling seamless integration into decentralized enterprise environments.
```
Comment (taskin0003) [inline index.md:15]: You should add/merge what you've deleted. It is our first release and introduces what Dent is

"In the first release, Arthur, DENT utilizes the Linux 5.6 Kernel to leverage switchDev, enabling the replacement of proprietary SDKs and NOSes with a standard open Linux interface. It also supports existing Linux toolchains."

File: index.md (hunk 1 above)
Comment (taskin0003) [inline index.md:17]: BTW, the link is wrong. The following is the right one
https://github.com/dentproject/dentOS/releases/tag/v2.0

File: index.md [hunk 4]
```diff
@@ -6,13 +6,40 @@ nav_order: 1
...
+
+In Q1 2022, the ["Beeblebrox" release (v2)](https://github.com/dentproject/dentOS/milestone/2) brought significant enhancements to DENT. Notable additions included NAT-44 and PT, NAT on pure Layer-3 port, support for configuring 802.1q interfaces, PoE controller integration, Dynamic Routing support, and Scale & Performance Improvements. These advancements fortified DENT's capabilities in handling diverse networking requirements with improved efficiency and scalability.
+
+The "Cynthia" release series marked a significant milestone for DENT. With the [v3.0 release](https://github.com/dentproject/dentOS/milestone/3) in Q1 2023, DENT introduced TC Persistence (Petunia), a new Kernel 5.15 (LTS), and support for IEEE 802.1x, QoS, IPv6, IGMP Snooping, and Egress Policer. These features enhanced DENT's security, performance, and compatibility, positioning it as a robust solution for distributed enterprise edge networking.
```
Comment (taskin0003) [inline index.md:19]: wrong link. It should have been
https://github.com/dentproject/dentOS/releases/tag/v3.0

**Review by taskin0003 — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (royalpinto007) [reply index.md:17]: According to the [roadmap](https://dent.dev/ecosystem/roadmap/), IPv6 is included in the third release and not in the second release, but there's no mention of it in the second release. Do you still want me to include it here?

Comment (taskin0003) [reply index.md]: And your link for the Beeblebrox link is wrong. The correct one is 
https://github.com/dentproject/dentOS/releases/tag/v2.0

File: index.md (hunk 2 above)
Comment (taskin0003) [reply index.md]: It is still wrong. Please make the following change
from 
    Additionally, the [v3.2 release](https://github.com/dentproject/dentOS/releases/tag/v3.1) in Q3 2023
to 
    Additionally, the [v3.2 release](https://github.com/dentproject/dentOS/releases/tag/v3.2) in Q3 2023

**Review by taskin0003 — CHANGES_REQUESTED** [review]
one more update. answered your questions.

File: index.md (hunk 1 above)
Comment (royalpinto007) [reply index.md]: fixed it here- https://github.com/dentproject/docs.dent.dev/pull/21/commits/69b9cdc5d1169e73ecd9e8394baed0297b36e63c

**Comment by royalpinto007** [comment]
I've squashed all 3 commits into a single one, as it's better to avoid having 3 commits here. I hope everything is resolved now.

**Review by taskin0003 — APPROVED** [review]

**Review by taraschornyiplv — APPROVED** [review]

**Pull request merged**